1/test/run

```diff
@@ -103,7 +103,7 @@ function test_get_job() {
 
 function run_tests() {
   local name=$1 ; shift
-  CONTAINER_ARGS="-u 10000:0"
+  CONTAINER_ARGS="-u 1001:0"
```

Comment:
(from your IRC query) we're setting it to 10000 to ensure it can run as a random uid, not the uid we built the image as.

if it only passes when you set the container uid to 1001, then it's going to fail when run on openshift.